kate-website2.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kate Harrison</title>
+    <link rel="stylesheet" href="reset.css">
+    <link rel="stylesheet" href="kate-website2.css">
+    <a name="top"></a> <!-- https://www.computerhope.com/issues/ch001475.htm#top -->
+</head>
+<body>
+    <!-- https://www.w3schools.com/howto/howto_js_topnav.asp -->
+    <!-- https://forum.freecodecamp.org/t/how-to-link-navbar-buttons-to-different-parts-of-a-website/201748 -->
+       <div class="header">
+          <div class="navbar">
+           <a class="active" href="#home">Home</a>
+            <a class="mystory" href="#mystory">About me</a>
+            <a href="#skills">Skills</a>
+            <a href="#hobbies">Hobbies</a>
+            <a href="#workexperience">Work experience</a>
+            <a href="https://github.com/search?q=katehrsn">Contact Me</a>
+            <a href="#top">Back to top</a>
+         </div>
+      </div>
+
+    <!-- https://www.w3schools.com/howto/howto_css_fixed_menu.asp -->
+
+  <main>
+        <div class="main">
+           <div class="title">
+               <h1>Kate <em>Harrison</em></h1>
+               <p>fashion designer.web developer.craftswoman</p>  
+           </div>
+          <div class="hero-image">
+                <div class="hero-text">
+                    <h2 style="font-size:50px">Hi, I am Kate!</h2>
+                    <h3>I like making things</h3>
+                    <h3>And meeting new people</h3>
+                </div>
+           </div>
+
+           <!-- https://www.w3schools.com/css/css_float_clear.asp -->
+           <section id="mystory">
```

Comment:
Keep consistent naming and case conventions.

```suggestion
           <section id="my-story">
```